test/MODCLUSTER-794/testit.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/sh
+
+# Shell to test MODCLUSTER-794
+
+pwd | grep MODCLUSTER-794
+if [ $? ]; then
+    PREFIX=MODCLUSTER-794
+else
+    PREFIX="."
+fi
+
+. includes/common.sh
+
+# first stop any previously running tests.
+tomcat_all_stop
+tomcat_all_remove
+httpd_all_clean
+
+MPC_NAME=MODCLUSTER-794 MPC_CONF=MODCLUSTER-794/mod_proxy_cluster.conf httpd_run
+
+for i in {1..20}; do
```

Comment:
Oh, almost forgot to mention, do we really need to boot 20 tomcats here @jajik ? Why not go with lower number like "3" and boot appropriate number of tomcats?

Reply:
I wanted to preserve the defaults, and resource-wise it didn't seem like a big issue. Of course, we can change it eventually.